tools/rand-seed/main.go

```diff
@@ -16,6 +17,18 @@ func main() {
 		return
 	}
 
-	fmt.Printf("base64:%s\n", base64.StdEncoding.EncodeToString(b))
-	fmt.Printf("base58:%s\n", base58.Encode(b))
+	// If the file doesn't exist, create it, or append to the file
+	f, err := os.OpenFile("random-seed.txt", os.O_WRONLY|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
```suggestion
	f, err := os.OpenFile("random-seed.txt", os.O_WRONLY|os.O_CREATE|os.O_APPEND , 0644)
```

Reply:
Actually here the comment is misleading. We do not want to append but to override what was in it before.